index.html

```diff
@@ -223,10 +221,10 @@ <h2 class="headline-content"><span>Use your favorite technology</span></h2>
       </div><!-- /.supported-technologies -->
 
       <p class="technology__description">
-        Use any language you like. Anything that works with heroku buildpacks can be used with the experimental platform. And since the interface to the hardware is via http connecting to it should work in any common language.
+        Use any language you like. Anything that works with heroku buildpacks can be used with the Experimental Platform. The interface to the hardware is http, so connecting to it should work in any common language.
       </p>
       <p class="technology__description">
-        This project wouldn't be possible without the extensive use of other great open source technologies: Docker, Dokku, Node.js, Go and so many more. The code is open source under Apache 2.0. Feel free to build, fork and share.
+        This project wouldn't be possible without the extensive use of other great open source technologies: Docker, Dokku, Node.js, Go and so many more.
```

Comment:
removed info for open source license. I think it's pretty clear when looking at the rest of the page that the project is open source. which license is being used should be clear once they look at the github repos.